app/models/annotation_text.rb

```diff
@@ -6,7 +6,7 @@ class AnnotationText < ActiveRecord::Base
   # AnnotationText is destroyed.
   has_many :annotations, dependent: :destroy
 
-  belongs_to :annotation_category
+  belongs_to :annotation_category, :counter_cache::true
```

Comment:
Er, so the correct syntax is actually `counter_cache: true`. What you have now is an error.